### DISCUSSION TITLE
Welcome to homebridge-zigbee-nt Discussions!

### DISCUSSION THREAD

**Discussion by madchicken** [madchicken/homebridge-zigbee-nt#50] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hughfr4nc15** [comment]
Does this need some kind of gateway/hub or (for ex.) IKEA Trådfri will work out of the box connecting to Homebridge?

**Reply by madchicken** [reply]
You need a ZigBee dongle, like the CC2531 USB

**Reply by hughfr4nc15** [reply]
Thanks.

**Reply by herod05** [reply]
any way to use the echo 4th gen (with its built-in Zigbee hub) insted of a ZigBee dongle?

**Comment by mojijafari** [comment]
dear Madchicken, I have drf1605h module. is there any way to connect it to homebridge? if yes can you explain the wiring and so on?

**Reply by madchicken** [reply]
Does it work with zigbee2mqtt? The plugin is based on the same library. Also, is the module available as an USB port? If so, I'd try to use the port address into the config (something like `/dev/USBxxxx`)

**Reply by madchicken** [reply]
I checked [this page](https://www.zigbee2mqtt.io/information/supported_adapters.html) and it seems your antenna is not supported

**Reply by mojijafari** [reply]
thanks for replay, it was very cool to connect with you!
I've connected to this module, I found the wiring and if you have a way to share it in zigbee2mqtt, I can send the datasheet and discription.
by the way, thanks for replay.
Could you please describe more about pairing new device in your code. I am amature and could understand your explanation.

**Reply by madchicken** [reply]
Yeah, I am expanding the wiki...I just have limited time

**Reply by madchicken** [reply]
@mojijafari could you check the wiki page https://github.com/madchicken/homebridge-zigbee-nt/wiki/How-to-support-new-devices and report some feedback? Thanks!

**Reply by mojijafari** [reply]
Dear madchichken,
This is very confiusing, could you please show the full sample or upload a movie from steps?

**Reply by madchicken** [reply]
@mojijafari The full sample is inside the wiki page (at the end). I can't upload a video, since it would require too much effort and time. I am sorry, but if you don't understand the process described in that page, I think you are missing some fundamental knowledge needed to implement the device adapter.

**Reply by mojijafari** [reply]
Yes, You are right, if you have any source that would be usefull for me, could you please share with me?

**Comment by petertmooney** [comment]
Hi there

Everything seems to be working fine, but being somewhat of a noob I'm not exactly sure what this means or which ping to install:
[ZigBee] Device TRADFRI LED bulb E27 WW clear 250 lumen, dimmable is a router, install ping

Can anybody please point me in the right direction?

**Reply by madchicken** [reply]
Routers are used by the network to expand the signal (ZigBee network is a mesh network) so it is important they stay alive, otherwise other devices in the network could become unresponsive

**Reply by petertmooney** [reply]
Thank you for the reply. 
how do I install ping though? Does it have to be a specific add on to work with yours, or do I just ignore it? Lol

**Reply by madchicken** [reply]
The plugin is doing ping every 30 seconds by default to all router devices unless you turn it off from the configuration, so you don't have to do anything

**Comment by herod05** [comment]
Any way to use the echo 4th gen (with its built-in Zigbee hub) instead of a ZigBee dongle?

**Comment by Figo66** [comment]
Hello, can you add or guide me in adding support for a new device ? Lidl ParkSide SmartHome Smart Watering Timer. Model HG06875. Thanks

**Reply by madchicken** [reply]
It seems the model is stil not supported by the zigbee herdsman library: https://github.com/Koenkk/zigbee2mqtt/issues/7695

**Comment by ft5221079** [comment]
Hello,

I have a WEB UI question that i can not find a answer.  

I have ZigBee dongle on my device. And i planed transfer to my other respberry pi 4 with home bridge installed.

question is i can not log in to WEB UI addrss.

Every time I got connection refused message when i connected

{
    "name": "ZigBee",
    "channel": 11,
    "secondaryChannel": "25",
    "adapter": "zstack",
    "routerPollingInterval": 30,
    "disableRouterPolling": false,
    "disableHttpServer": false,
    "httpPort": 9191,
    "disableLed": false,
    "preferAutoDiscover": true,
    "platform": "ZigbeeHomebridgeNTPlatform"
}

**Reply by madchicken** [reply]
Could you post logs of the startup of homebridge? Please, enable debug from the settings section in the homebridge web Ui
